web/styles/typing_notifications.css

```diff
@@ -9,3 +9,9 @@
     list-style: none;
     margin: 0;
 }
+
+/*  override margine-bottom of main-view-banner since it causes gap
+    between mark as read banner and typing notification.  */
+#mark_as_read_turned_off_banner {
+    margin-bottom: 0 !important;
```

Comment:
You don't need to use `!important` over here, the specificity of ID > Classes. Also, let's do `margin: 5px 0` instead, the typing notification feels too close to the banner.

Reply:
done!

Reply:
The comment should no longer say `margin-bottom` and let's bring this comment just above the margin property since we are talking about that property specifically.